pkg/config/validator.go

```diff
@@ -490,7 +491,11 @@ func convertAspectParams(f AspectValidatorFinder, name string, params interface{
 	if err := decode(params, ap, strict); err != nil {
 		return nil, ce.Appendf(name, "failed to decode aspect params: %v", err)
 	}
-	if err := avl.ValidateConfig(ap, expr.NewCEXLEvaluator(), df); err != nil {
+	eval, err := expr.NewCEXLEvaluator(1024)
```

Comment:
@mandarjog @ZackButcher : Do you guys think we can/should reuse the same evaluator constructed during the server init ? The hard coded number 1024 here is kind of bothersome.